clients/src/main/java/org/apache/kafka/clients/ClientRequest.java

```diff
@@ -87,11 +87,11 @@ public boolean isInitiatedByNetworkClient() {
         return isInitiatedByNetworkClient;
     }
 
-    public long getSendMs() {
+    public long sendTime() {
         return sendMs;
     }
 
-    public void setSendMs(long sendMs) {
+    public void setSendTime(long sendMs) {
```

Comment:
So, is the suggestion for the other methods to be `creationTimeMs` and `receivedTimeMs`?